README.md

```diff
@@ -1,5 +1,10 @@
 # SeqAn3 -- the modern C++ library for sequence analysis
 
+[![build status](https://travis-ci.org/seqan/seqan3.svg?branch=master)](https://travis-ci.org/seqan/seqan3)
+[![codecov](https://codecov.io/gh/seqan/seqan3/branch/master/graph/badge.svg)](https://codecov.io/gh/seqan/seqan3)
+[![latest release](https://img.shields.io/github/release/seqan/seqan3.svg)](https://github.com/seqan/seqan3/releases/latest)
+[![platforms](https://img.shields.io/badge/platform-linux%20%7C%20bsd%20%7C%20osx-informational.svg)](https://github.com/seqan/seqan3/releases/latest)
```

Comment:
Very nice, thanks for adding this.

I would remove platforms and add the following:
```c++
[![license](https://img.shields.io/badge/license-BSD-green.svg)](https://github.com/seqan/seqan3/LICENSE.md)
![star](https://img.shields.io/github/stars/seqan/seqan3.svg?style=social)
![follow](https://img.shields.io/twitter/follow/SeqAnLib.svg?label=follow&style=social)
```

Also I would move it over the head line maybe?

Reply:
@h-2 See the [rich-diff](https://github.com/seqan/seqan3/pull/1086/files?short_path=04c6e90#diff-04c6e90faac2675aa89e2176d2eec7d8). I think this does not look good.

Reply:
Hm, I still think it looks better above. It's not really part of the decription, it's "meta" information or not?

Reply:
Okay let's try it out. I find it better if the badges come after the headline, since the headline (usually the project name) shows that the badges belong to the project.

Here some examples :)

https://github.com/doxygen/doxygen after the headline
https://github.com/google/googletest after the headline
https://github.com/aterris/simplecov-shield after the headline
https://github.com/dotnet/try after the headline
https://github.com/danger/danger after the headline
https://github.com/dwyl/learn-travis after the headline
https://github.com/HariSekhon/Nagios-Plugins after the headline

https://github.com/pyupio/safety after the logo
https://github.com/tensorflow/tensorflow after the logo

-----

https://github.com/goreleaser/goreleaser in the text
https://github.com/haskell/cabal in the headline << I like how the logo is right-aligned :)
https://github.com/travis-ci/travis.rb in the headline
https://github.com/bamlab/generator-rn-toolbox in the headline

------

https://github.com/beevelop/docker-shields before the headline
https://github.com/tox-dev/tox before the headline